https option and landing on all posts page

emc2cube opened the pull request:
Updates to add an option to use https (default is yes) and land directly on the "All posts" page when button is clicked.

emc2cube committed:
Add https option

emc2cube committed:
Add https option

knyar commented:
Could you please implement this without changing current defaults?
- keep http as the default schema. The Old Reader is mostly used to read public content, so we would prefer not to slow things down by another 2 RTTs for SSL handshake. Those users who need https can explicitly enable it.
- make "All posts" a separate option. Feel free to take a look at how it's done in Chrome extension: https://github.com/knyar/theoldreader-chrome

emc2cube committed:
Add landing page options, refresh rate options. https now off by default

emc2cube committed:
Add landing page and refresh rate options

emc2cube commented:
Ok should be working:
- https is now off by default.
- Added an option to choose the landing page when the button is pressed, default is the home page.
- Added an option to change the refresh rate of the counter, default is still 10s.

emc2cube committed:
Wrong refresh. Fixed and new default updated to 15 minutes

emc2cube committed:
Fixed: wrong refresh counter.

krasnoukhov commented:
Thanks!
